admin/config-ui/class-configuration-page.php

```diff
@@ -234,4 +234,22 @@ private function remove_notification_option() {
 	private function get_options() {
 		return get_option( 'wpseo' );
 	}
+
+	/**
+	 * Returns the page identifier for the configuration UI.
+	 *
+	 * @return string The configuration page identifier.
+	 */
+	public static function get_configuration_page_identifier() {
+		return self::PAGE_IDENTIFIER;
+	}
+
+	/**
+	 * Returns the page identifier for the wpseo dashboard.
+	 *
+	 * @return string The dashboard page identifier.
+	 */
+	public static function get_wpseo_dashboard_page() {
```

Comment:
I think `get_admin_page_identifier` would be a better name and is more consistent with `get_configuration_page_identifier`